how_to_create_context_VTEX_IO.md

```diff
@@ -4,14 +4,56 @@ This article begins with a simple question, how is it possible to share data bet
 
 To give a little context the Sony's development team had to connect two blocks.
 
-[To connect blocks](/arquivos/to_connect_blocks.png)
+![To connect blocks](/arquivos/to_connect_blocks.png)
 
 For previous decisions these blocks were separated in different rows wich difficulted connecting the blocks.
 
 We thought that to use react context was a good idea but here is where appears the question, how to do it?
 
 The first issue, it is understand how react context work, it should have a provider wich to wrap the children and also it will keep the context values.
 
+## STEPS 
+
+It is important to know about react context if you do not know it, you can review the next links:
+
+- [Official documentation](https://reactjs.org/docs/context.html#gatsby-focus-wrapper)
+- [A Guide to React Context and useContext() Hook](https://dmitripavlutin.com/react-context-and-usecontext/)
+### STEP ONE
+
+We should to create the next files into custom app: 
+
+![files](/arquivos/files.png)
+
+Each file has a special function, we will begin speaking of `context.tsx` file.
```

Comment:
"speaking of `context.tsx` file" is lacking a "the" between "of" and `context.tsx`